pom.xml

```diff
@@ -4,20 +4,20 @@
   <parent>
     <groupId>org.nuxeo</groupId>
     <artifactId>nuxeo-parent</artifactId>
-    <version>11.4-SNAPSHOT</version>
+    <version>2021.0-SNAPSHOT</version>
   </parent>
 
   <groupId>org.nuxeo.retention</groupId>
   <artifactId>nuxeo-retention-parent</artifactId>
   <name>Nuxeo Retention Management Parent</name>
-  <version>11.0.0-SNAPSHOT</version>
+  <version>2021.0-SNAPSHOT</version>
   <packaging>pom</packaging>
   <description>The Nuxeo Retention Management addon adds the capability to create and attach retention rules to documents
     in order to perform advanced record management.
   </description>
 
   <properties>
-    <nuxeo.target.version>11.*</nuxeo.target.version>
+    <nuxeo.target.version>lts-2021.*</nuxeo.target.version>
```

Comment:
I think here we have to specify ` 2021.*` as the LTS version is `2021.HFDigit.BuildDigit` I will test locally

Reply:
let me know if it's work because of the WebUI side we have this

![Screenshot 2020-12-02 at 14 28 46](https://user-images.githubusercontent.com/59877775/100878601-b161e680-34aa-11eb-998c-dc0d050c500a.png)